test/core/Leaf.t.sol

```diff
@@ -272,7 +285,7 @@ contract TickAndBinTest is Test {
     showTickApprox(1 ether, 1 ether);
   }
 
-  // int constant min_tick_abs = int(2**(BIN_BITS-1));
+  // int constant min_tick_abs = int(2**(TICK_BITS-1));
```

Comment:
This comment is a bit weird - there's no mention of ticks in this test?

Reply:
fixed in 1a4f5ad2db572b5b7ca732e7585ab93c0bc6c5a7